README.md

```diff
@@ -67,7 +69,13 @@ To download the code you can either go to the source code page and click `Code -
 
 Installing package from source is done using command line:
 
-> \$ python setup.py install
+> \$ python3 -m pip install .
```

Comment:
Good point! Thank you for point out that `setup` hasn't been supported for a while.